Backend/Middlewares/Clientlogger.js

```diff
@@ -1,11 +0,0 @@
-const Clientlogger = (req, res, next) => {
-  const { clientusername, clientfullname, clientemail, tags } = req.body;
-
-  if (clientusername && clientfullname && clientemail && tags) {
-    next();
-  } else {
-    return res.status(511).send("All fields are required");
```

Comment:
having validation in the backend is good.
it is not really needed if the front end validates it.